plugins/generator-1.19.4/forge-1.19.4/mappings/screens.yaml

```diff
@@ -2,6 +2,9 @@ _default: Ingame
 Ingame: Ingame
 MainMenu: TitleScreen
 IngameMenu: PauseScreen
+AbstractCommandBlock: AbstractCommandBlockEditScreen
+AbstractContainer: AbstractContainerScreen
+AbstractFurnace: AbstractFurnaceScreen

```

Comment:
AbstractSignEditScreen also exists in 1.19.4

Reply:
:heavy_check_mark: Fixed